src/main/java/com/github/dockerjava/api/command/CreateContainerCmd.java

```diff
@@ -159,16 +159,16 @@
      * @since 1.19
      */
     @CheckForNull
-    Boolean isOomKillDisable();
+    Boolean getOomKillDisable();
 
     @CheckForNull
-    Boolean isPrivileged();
+    Boolean getPrivileged();
 
     @CheckForNull
-    Boolean isPublishAllPorts();
+    Boolean getPublishAllPorts();
 
     @CheckForNull
-    Boolean isReadonlyRootfs();
+    Boolean getReadonlyRootfs();
 
     @CheckForNull
     Boolean isStdInOnce();
```

Comment:
I made docker-java generator for getters/setters/tostring/hash/equals and used them for only new fields. But then decided to generate all of them and fixed places that used to return boolean for Boolean...